src/sidebar/config/fetch-config.js

```diff
@@ -212,11 +212,18 @@ export async function fetchConfig(appConfig, window_ = window) {
       requestConfigFromFrame.ancestorLevel,
       window_
     );
-    return await fetchConfigRpc(
+
+    const rpcMergedConfig = await fetchConfigRpc(
       appConfig,
       parentFrame,
       requestConfigFromFrame.origin
     );
+    // Add back the optional focused group id from the host page config
+    // as this may be needed for additional apps such aas the notebook.
```

Comment:
```suggestion
    // as this is needed in the Notebook
```

This behavior is really only for the Notebook at present, so I think we should just say that.